valhalla/baldr/pathlocation.h

```diff
@@ -167,7 +173,8 @@ struct PathLocation : public Location {
 
   static Location fromPBF(const odin::Location& loc) {
     Location l({loc.ll().lng(), loc.ll().lat()},
-               odin::Location::kBreak ? Location::StopType::BREAK : Location::StopType::THROUGH,
```

Comment:
just for posterity, this is where the real bug lies. kBreak == 0 and therefore everything gets set to through. im pretty sure that this bug is my fault.. sorry all!